dri3_ws.c

```diff
@@ -306,7 +306,6 @@ static void destroy_buffer(struct driws_buffer *buffer)
 #endif
 
 #ifdef DRI3WS_USE_DUMB
-	close(buffer->dmabuf_fd);
```

Comment:
Good catch. While at it, perhaps change the `buffer->dmabuf_fd = 0` above to `buffer->dmabuf_fd = -1`. -1 is a better invalid number for the fd.

Reply:
Fixed

Reply:
Fixed